src/Message/CIMResponse.php

```diff
@@ -7,6 +7,11 @@
  */
 class CIMResponse extends CIMAbstractResponse
 {
+    /**
+     * For Error codes: @see https://developer.authorize.net/api/reference/responseCodes.html
+     */
+    const ERROR_RESPONSE_CODE_CANNOT_ISSUE_CREDIT = 54;
```

Comment:
> Which other error codes?

`CIMGetProfileResponse::ERROR_DUPLICATE_PROFILE`, `CIMGetProfileResponse::ERROR_MAX_PAYMENT_PROFILE_LIMIT_REACHED`

> I rather keep error codes as constants

My comment was never against constants; it was to keep the values of these error constants consistent i.e. as strings.